### PR TITLE
ZENKO-815 doc

### PR DESCRIPTION
**Pull request by wabernat**
This is the last of three versions of the Installation Guide, superseding Zenko-655 and -1057.

### CODE REVIEW

File: docs/Installation/Installation_Guide.rst [hunk 1]
```diff
@@ -21,74 +21,68 @@ public cloud resources, the following sizing requirements are assumed good for
...
 
-
 Sizing
 ======
 
-Sizing for Metal K8s
---------------------
-
--  Cores
+The following sizes have been for Zenko instances tested on live systems using
```
Comment (dora-korpar) [inline docs/Installation/Installation_Guide.rst]: "The following size for Zenko instances..."? Sounds kinds weird the way it is

File: docs/Installation/Installation_Guide.rst [hunk 2]
```diff
@@ -102,315 +96,29 @@ If you are behind a proxy, add the following lines to your local machine’s
     https_proxy=http://user;pass@<my-ip>:<my-port>
     no_proxy=localhost,127.0.0.1,10.*
 
-******************************
```
Comment (dora-korpar) [inline docs/Installation/Installation_Guide.rst:105]: Is all this deleted stuff getting moved the metalk8s repo?

Comment (wabernat) [reply docs/Installation/Installation_Guide.rst:105]: That is my hope.

File: docs/Installation/Installation_Guide.rst (hunk 1 above)
Comment (wabernat) [reply docs/Installation/Installation_Guide.rst]: Good catch.

**Review by tmacro — APPROVED** [review]

File: docs/Installation/Installation_Guide.rst [hunk 3]
```diff
@@ -21,74 +21,68 @@ public cloud resources, the following sizing requirements are assumed good for
...
-   -  4 for node
+Reserve the following resources for each node.
 
--  RAM
+-  Cores per server (2 vcpu = 1 core)
 
-   -  4 GB for etcd
-   -  8 GB for master
-   -  8 GB for node
+   - 12 cores minimum
```
Comment (anurag4DSB) [inline docs/Installation/Installation_Guide.rst]: These are vCPU and should be 
24 vCPU minimum
32 vCPU medium load
58 vCPU heavy load

Reference: https://docs.google.com/presentation/d/1cKNQKa5YlnCKYIOjCOy4KeM9oZbWlLuAZbxCTF2uqN0/edit#slide=id.g3fd7445d35_1_5

**Review by giacope — DISMISSED** [review]
Please have more meaningful commit messages. I would recommend squashing all your commits and writing a commit message that summarizes the ticket given. Also, try to write it using imperative tense (or at least past tense).

**Review by anurag4DSB — CHANGES_REQUESTED** [review]

File: docs/Installation/Installation_Guide.rst [hunk 4]
```diff
@@ -21,74 +21,68 @@ public cloud resources, the following sizing requirements are assumed good for
...
+      **Note:** This requirement is for storage, not for the system device.
+      You may have to attach a separate storage volume to your cloud server
+      instance. Storage volumes must match or exceed the maximum anticipated
+      demand. Once set, the cluster cannot be resized without redefining new
+      volumes.
 
-All servers must be in CentOS 7.4, and accessible with ssh.
+All servers must be in CentOS 7.4 or later, and must be ssh-accessible.
 
 Proxies
```
Comment (giacope) [inline docs/Installation/Installation_Guide.rst:87]: I still don't know where these proxy settings come from 🤔

File: docs/Installation/Installation_Guide.rst (hunk 3 above)
Comment (wabernat) [reply docs/Installation/Installation_Guide.rst]: changed and pushed.

Comment (anurag4DSB) [reply docs/Installation/Installation_Guide.rst]: Or rather get the cores too from the link

**Review by anurag4DSB — APPROVED** [review]

**Review by ssalaues — APPROVED** [review]

**Comment by ssalaues** [comment]
@bert-e wait

**Pull request merged**